spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/cassandra/CassandraMetricsAutoConfiguration.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2012-2019 the original author or authors.
```

Comment:
Any reason why the copyright ends at 2019 instead of 2020?

Reply:
oversight, from a copy-paste